examples/tutorials/tutorial_BET_Disk.py

```diff
@@ -0,0 +1,161 @@
+import flow360 as fl
+import flow360.component.simulation.units as u
+from examples.migration_guide.bet_disk import bet_disk_convert
+from flow360.component.simulation.unit_system import SI_unit_system
+from flow360.examples import Tutorial_BETDisk
+
+"""
+In this tutorial case we are using Blade Element Theory (BET) to simulate the XV-15 rotor blade. Documentation for this tutorial is available in the link below.
+
+https://docs.flexcompute.com/projects/flow360/en/latest/tutorials/BETTutorial/BETTutorial.html
+"""
+
+
+fl.Env.preprod.active()
+
+project = fl.Project.from_file(Tutorial_BETDisk.geometry, name="Tutorial BETDisk from Python")
+# project = Project.from_cloud(project_id="prj-edc9c40f-1de9-4a7d-ac64-122a483609dc")
+geometry = project.geometry
+
+# show face and edge groupings
+geometry.show_available_groupings(verbose_mode=True)
+geometry.group_faces_by_tag("faceName")
+geometry.group_edges_by_tag("edgeName")
+
+# ask about whether to define inner_radius since by default it is assigned None and not 0
+
+with SI_unit_system:
+    cylinder1 = fl.Cylinder(
+        name="cylinder1",
+        axis=[1, 0, 0],
+        center=[-2.0, 5.0, 0],
+        outer_radius=4.0,
+        inner_radius=0,
+        height=0.6,
+    )
+    cylinder2 = fl.Cylinder(
+        name="cylinder2",
+        axis=[1, 0, 0],
+        center=[0, 5, 0],
+        outer_radius=4.1,
+        inner_radius=0,
+        height=5,
+    )
+    BETDisks, Cylinders = bet_disk_convert(
+        file=Tutorial_BETDisk.case_json, save=True, omega_unit=u.flow360_angular_velocity_unit
+    )
```

Comment:
I think we want so separate these 2 things. Here, use already converted BET. Add another example in migration guide and call this function to get BET input